src/index.html

```diff
@@ -2,21 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Stop watch</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
The href attribute points to a '.scss' file. Ensure that this file is being processed into CSS, as browsers cannot directly interpret SCSS files. If the SCSS is already compiled, consider linking to the resulting '.css' file instead.